frontend/lib/norent/letter-builder/confirmation.tsx

```diff
@@ -47,19 +70,23 @@ export const NorentConfirmation: React.FC<{}> = () => {
         documentation as you can. This can include a letter from your employer,
         receipts, doctor’s notes etc.
       </p>
-      {stateName && (
-        <>
+      <>
+        {stateName && needsDocumentation && (
           <p>
```

Comment:
@toolness am I generally being too careful here in not rendering these elements without these values being defined? Is this something I should let typescript keep an eye on instead?

Reply:
ahh yes definitely let typescript keep an eye on these instead!  i intentionally added a bunch of tests to `national-metadata.test.tsx` in #1268 to make sure that all our assumptions about the data, which are encoded into the typescript typings, are actually correct!